src/less/modules/breadcrumbs.less

```diff
@@ -1,7 +1,9 @@
 .breadcrumb {
-  margin-left: 15px;
-  margin-top: -20px;
+  //TODO move breadcrumbs out into app-wide directive under navbar not within ui-view
```

Comment:
That's what I meant, I noticed ui-router is included but (at least at face-value) it doesn't seem to be doing much more than what you'd get out of regular angular-route. I might try and start on implementing some of the stuff like subviews over the next couple of weeks.
Even more so than this bootstap stuff, it's going to touch pretty much everything in the app so it's going to be fairly involved process.